### ISSUE TITLE
Missing links to NS-Forest markers in KG

### ISSUE THREAD

**Issue by dosumis** [Cellular-Semantics/whole_human_brain_ontology#15]
For some reason, has_part links to NS Forest markers are not showing up in KG

<img width="530" alt="image" src="https://github.com/hkir-dev/whole_human_brain_ontology/assets/112839/d12ac840-6159-4726-bba7-a3ee1bc528bd">

<img width="578" alt="image" src="https://github.com/hkir-dev/whole_human_brain_ontology/assets/112839/cd8e56bb-6f4f-408e-982c-0c323ef63c6c">

<img width="537" alt="image" src="https://github.com/hkir-dev/whole_human_brain_ontology/assets/112839/e98234e8-c638-4ad6-b3a8-10ccbf66d432">

**Comment by dosumis** [comment]
I'm wondering whether it's because they are all in one axiom, rather than 3 separate - even though these are semantically identical.

**Comment by hkir-dev** [comment]
Yes, this was an neo4j2owl issue we faced in bdso. As a workaround, we added a Sparql query that expands intersectionOf definitions https://github.com/obophenotype/bds-pipeline-config/blob/dev/config/dumps/sparql/construct_subclass_of_intersection.sparql

I will add this query to OBASK as quick fix then, I will investigate [neo4j2owl issue](https://github.com/VirtualFlyBrain/neo4j2owl/issues/82) in detail.

**Comment by hkir-dev** [comment]
https://github.com/hkir-dev/whbo_pipeline updated to fix the issue.
![image](https://github.com/hkir-dev/whole_human_brain_ontology/assets/73784267/fb935dd2-3872-472f-a6ea-007e341ff2f2)
